app/database/MetricsDB.scala

```diff
@@ -21,8 +23,10 @@ class MetricsDB(val db: Database) {
   def getForks: Seq[Fork] = await(db.run(forksTable.result))
   def insertFork(fork: Fork): Int = await(db.run(forksTable += fork))
 
+  val dayOfWeek = SimpleFunction.binary[String, Timestamp, Timestamp]("date_trunc")
```

Comment:
`date_trunc` does some rounding, is the result of `date_trunc('date', 2017-01-01T23:59:59)` correct for this use case?